README.md

```diff
@@ -130,6 +130,18 @@ You can deploy this project using your own deployment solution
 - Twitter: [@dandrianarivo](https://twitter.com/dandrianarivo)
 - LinkedIn: [LinkedIn](https://linkedin.com/in/andrianarivo)
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you made the commit message capitalized to make your pull request professional and descriptive 😊